tests/AutoReview/ProjectCodeTest.php

```diff
@@ -110,7 +110,7 @@ static function (string $className): bool {
      *
      * @param class-string $className
      *
-     * @requires PHP 8.2
+     * @requires PHP ^8.2
```

Comment:
```suggestion
     * @requires PHP >=8.2
```

probably, as `caret` will not allow PHP 9.0 (if caret is defined the same way as in composer)

Reply:
Indeed, `8.0` from annotation should change to `>= 8.0` in attribute.